packages/ui-components/src/__tests__/DeploymentSteps.test.ts

```
@@ -635,28 +640,40 @@ const defaultProps: DeploymentStepsProps = {
 	});
-
-	const setGui = (defaultProps: DeploymentStepsProps) => {
-		defaultProps.gui = {
-			areAllTokensSelected: vi.fn(),
-			getSelectTokens: () => [],
-			getNetworkKey: vi.fn(),
-			getCurrentDeployment: () => mockDeployment,
-			getAllFieldDefinitions: () => []
-		} as unknown as DotrainOrderGui;
-	};
+	let mockGui: DotrainOrderGui;
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'mockGui' is only assigned here.


(lint/style/useConst)